src/bindgen/call_builder/crystal_from_cpp.cr

```diff
@@ -6,7 +6,7 @@ module Bindgen
       end
 
       # Calls the *method*, using the *proc_name* to call-through to Crystal.
-      def build(method : Parser::Method, receiver = "self") : Call
+      def build(method : Parser::Method, receiver = "self", do_block = false) : Call
```

Comment:
What does `do_block` do?

Reply:
It generates snippets like `Proc(Void).new do ... end` instead of `Proc(Void).new { ... }`. This is because the template will be used inside the `BindgenHelper.wrap_proc(%)` template, which now comes directly from `builtin_types.yml`, and is a full template string that has access to environment variables. Thus `Util.template` will interpret the `Proc` body as an environment variable, unless the `do` notation is used.

Other solutions are to disable ENV access from `from_crystal` and friends, or support escape sequences for braces in `Util.template`.

Reply:
Ah ok, could you put what you just said into the docs of the method?  Escaping-Support would be great, but that's for another day.